tests/flow_proto_props/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(__dirname, ["flow"]);
```

Comment:
Since this is supported by babel, too, should we run this with babylon as well?

Reply:
We actually _just_ landed it, so I'll enable it after our next release!

Reply:
Cool :+1: